### PR TITLE
Rework runner to a more simplistic docker-compose based approach

### PR DESCRIPTION
**Pull request by brandonwestcott**
Kind of a big change in direction here, but I think this supports our needs and reduces having to manage a "codebase"

The idea here is to give the user a 'simple' local network and allow them to use the normal `docker-compose` to run their tests, rather than relying on custom code & config.

**Its probably easier to view this PR just as a "new" project: https://github.com/quorumcontrol/tupelo-integration-runner/tree/docker-compose-based**

Functionally all this does is create a docker image that contains a [full tupelo network docker-compose](https://github.com/quorumcontrol/tupelo-integration-runner/blob/docker-compose-based/docker/docker-compose.yml) with the required [toml configs](https://github.com/quorumcontrol/tupelo-integration-runner/tree/docker-compose-based/docker/config) and a small [runner](https://github.com/quorumcontrol/tupelo-integration-runner/blob/docker-compose-based/docker/run.sh). Then a user can run this image in their normal `docker-compose.yml`, then on start, this container will inspect its environment (by mounting `/var/run/docker.sock`), and will run a tupelo network thats accessible by the other containers in the parent stack.

See https://github.com/quorumcontrol/tupelo-integration-runner/blob/docker-compose-based/README.md for a bit more description on usage.

### CODE REVIEW

**Comment by brandonwestcott** [comment]
Also I think maybe renaming this from `tupelo-integration-runner` to `tupelo-local` would make sense.

File: docker/config/bootstrap.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Port = 34001
+BootstrapOnly = true
+NotaryGroupConfig = "./notarygroup.toml"
+
```
Comment (tobowers) [inline docker/config/bootstrap.toml:4]: Should boostrap have an IP config?

**Review by tobowers — APPROVED** [review]
looks awesome! I like the simplicity.

**Comment by brandonwestcott** [comment]
Closed in favor of https://github.com/quorumcontrol/tupelo-local/pull/1

**Pull request closed without merging**